Dockerfile.x86

```diff
@@ -1,53 +1,57 @@
-# docker build -t docker.io/robertbartlettbaron/acct-req .
-# docker run docker.io/robertbartlettbaron/acct-req
-# docker push "docker.io/robertbartlettbaron/acct-req"
-#FROM registry.redhat.io/rhel  
-FROM centos
+# FROM registry.redhat.io/rhel  
+# FROM centos7
+FROM fedora:30
 
-WORKDIR /app
-
-RUN yum install -y cpan
-
-RUN yum list | grep python
+# this is for fedora:30 power9
+# Bug https://bugzilla.redhat.com/show_bug.cgi?id=1694411
+#RUN echo "zchunk=False" >> /etc/dnf/dnf.conf && \
+#    dnf install -y git gcc httpd python3 python3-pip python3-devel \
+#        python3-mod_wsgi mod_auth_openidc libffi-devel openssl-devel
 
-#RUN yum -y install rh-python36
-#RUN yum -y install rh-hython36-pip 
+WORKDIR /app
 
-RUN yum -y install make gcc libffi-devel openssl-devel bzip2-devel wget wireshark git python36.x86_64 python36-devel.x86_64\
-  && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py \
-  && python3 get-pip.py
 
-RUN python3 --version
-# && pip3.4 install --upgrade pip \
-# && pip3.4 install 'setuptools>=18.5' \
-# && cd /app
- 
-RUN mkdir /app/openshift-acct-mgt
-#  && cd /app/openshift-acct-mgt \
-#  && pip3 install gunicorn \
-#  && pip3 install Flask \
-#  && pip3 install kubernetes \
-#  && pip3 install openshift
+RUN rpm --initdb \
+ && yum clean all \
+ && yum -y install make gcc libffi-devel openssl-devel bzip2-devel wget wireshark git python3 httpd httpd-devel httpd-tools  python3-devel python3-pip \ 
+ && pip3 install --upgrade pip \
+ && pip3 install setuptools \
+ && pip3 install mod_wsgi \
+ && pip3 install Flask-HTTPAuth \
+ && yum install -y cpan
```

Comment:
Handle Python dependencies using `requirements.txt` rather than installing them individually with `pip` like this. Having everything in `requirements.txt` makes it much easier for someone else to test your code out locally. Dependencies in `requirements.txt` can be installed by running `pip3 install -r requirements.txt`.